carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carla_to_carma_external_objects

```diff
@@ -63,7 +65,26 @@ def converter(object):
     object_msg.header = object.header
     object_msg.id = object.id
     object_msg.pose.pose = object.pose
-    object_msg.velocity.twist = object.twist
+
+    ## CARLA ROS Bridge will not provide the vehicle twist info which generated,
+    ## As a result, we need to calculate that based on position and time differences
+    if object.id in prev_objects:
+        ## time diff
+        curr_time = object.header.stamp.secs + 10**-9 * object.header.stamp.nsecs
+        prev_time = prev_objects[object.id]['object'].header.stamp.secs + 10**-9 * prev_objects[object.id]['object'].header.stamp.nsecs
+        time_diff = curr_time - prev_time
```

Comment:
Minor detail, but is it conceivable we might get 2 updates in the same simulation timestep? Might be worth trying to gracefully handle the divide-by-zero case here somehow if so, but only if its possible we'd get two same-timestep updates.